OpenEdXMobile/build.gradle

```diff
@@ -426,6 +426,8 @@ android {
          */
         //TODO: Remove GradleCompatible after we upgrade to Android Studio 2.4
         disable 'GradleCompatible'
+        disable 'MissingTranslation'
+        disable 'ImpliedQuantity'
```

Comment:
What's the reason of disabling this lint check?

Reply:
@miankhalid In plural string french language standards are different then others so it cause an error of Implied quantity that's why we disable it

Reply:
Ok, it makes sense to mark this as a warning too instead of disabling the check. Which has been done in 501ccfd 👍